packages/eslint/src/rules/html/migrations/no-rounded-borders.ts

```diff
@@ -0,0 +1,60 @@
+import { createRule } from '../../../utils/create-rule';
+import { HtmlNode } from '../../../parsers/html/html-node';
+
+export const name = 'no-rounded-borders';
```

Comment:
Not sure if that name is describing what's going on here. To me, this would rather be something like "no-unnumbered-border-radius". What do you think?